openhands/server/session/agent_session.py

```diff
@@ -331,7 +331,7 @@ async def _create_runtime(
         if runtime_cls == RemoteRuntime:
             # If provider tokens is passed in custom secrets, then remove provider from provider tokens
             # We prioritize provider tokens set in custom secrets
-            provider_tokens_without_gitlab = (
+            provider_tokens_without_gitlab_bitbucket = (
```

Comment:
```suggestion
            overrided_tokens = (
```

i think the agent got confused by the existing var name